docker_tests/docker_tests_utils.py

```diff
@@ -0,0 +1,115 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+import os
+import subprocess
+from pathlib import Path
+from typing import List
+
+docker_image = os.environ.get('DOCKER_IMAGE')
```

Comment:
There is no need to use env variable, I think for python tool like that it would be much bettter to use `click` and pass the image as command line option.

Reply:
This is a variable is used by the test so we can't use `click`. I mimic the other tests that pass parameters for tests using environment variables.  For example, https://github.com/apache/airflow/blob/8505d2f0a4524313e3eff7a4f16b9a9439c7a79f/tests/providers/apache/hive/sensors/test_metastore_partition.py#L28  and any system tests for Google operator.